recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.5.2" %}
+{% set version = "1.6.0" %}
 {% set name = "JPype1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.lower() }}-{{ version }}.tar.gz
-  sha256: 74a42eccf21d30394c1832aec3985a14965fa5320da087b65029d172c0cec43b
+  url: https://github.com/jpype-project/jpype/archive/refs/tags/v{{ version }}.zip
```

Comment:
According to conda-forges rules we should stick to PyPI sources. The archives on Github could change in principle.